course_info/templates/course_info/partials/people.html

```diff
@@ -1,10 +1,10 @@
 {% verbatim %}
 <nav>
-  <h1>
-    <a href="#/">Find Course</a>
+  <h3>
+    <a onclick="window.history.go(-2); return false;">Admin Console</a> > <a href="#/">Find Course</a>
```

Comment:
This will get you to the right page, but it screws with the user's expectations of how their browser history works.  Ideally we want to retrieve the URL from the session history, and then navigate to it.